.github/workflows/build.yml

```diff
@@ -188,6 +188,7 @@ jobs:
           PKG_CONFIG_ALLOW_CROSS: 1
           PKG_CONFIG_LIBDIR: /usr/lib/${{ matrix.pkgconfig }}/pkgconfig
           TARGET: ${{ matrix.target }}
+          JEMALLOC_SYS_WITH_LG_PAGE: 16
```

Comment:
Since your adding it into the cross part of the build script it would effect apple silicon macs, you should add it to the native too

Line 188:
```yaml
        # pkg-config --variable pc_path pkg-config
         cargo  build --release --all-features
```

To

```yaml
        # pkg-config --variable pc_path pkg-config
        export JEMALLOC_SYS_WITH_LG_PAGE=16
        cargo  build --release --all-features
```